app/form/v1_0/steps/drug-use.ts

```diff
@@ -108,7 +109,12 @@ const sectionConfig: SectionConfig = {
         drugsUseFields.sectionComplete(),
       ].flat(),
       template: templates.drugUsageNew,
-      next: stepUrls.drugUseHistory,
+      next: [
```

Comment:
I think this needs adding to steps `stepUrls.drugDetailsInjected` and `stepUrls.drugDetails` too.